doc/man3/EC_GROUP_copy.pod

```diff
@@ -181,9 +181,17 @@ EC_GROUP_method_of returns the EC_METHOD implementation in use for the given cur
 
 EC_GROUP_get0_generator returns the generator for the given curve or NULL on error.
 
-EC_GROUP_get_order, EC_GROUP_get_cofactor, EC_GROUP_get_curve_name, EC_GROUP_get_asn1_flag, EC_GROUP_get_point_conversion_form
-and EC_GROUP_get_degree return the order, cofactor, curve name (NID), ASN1 flag, point_conversion_form and degree for the
-specified curve respectively. If there is no curve name associated with a curve then EC_GROUP_get_curve_name will return 0.
+EC_GROUP_get_order() returns the order for the specified curve.
```

Comment:
This is not exact technically.

The signature of the function is
~~~c
int EC_GROUP_get_order(const EC_GROUP *group, BIGNUM *order, BN_CTX *ctx);
~~~

So the return value will be an `int` not really the `order`, which is instead `BN_copy`-ed in the second argument (`BIGNUM *order`).

Ideally we should check the implementation of `EC_GROUP_get_order()` and check what are the actual `int` return values when everything goes well and when something fails (possibly documenting what happens to the "output" variable, i.e. `order`, in the latter cases).

This also applies to other functions in this PR.